leakcanary-watcher/src/main/java/com/squareup/leakcanary/LeakTraceElement.java

```diff
@@ -176,6 +177,20 @@ public String toString(boolean maybeLeakCause) {
       string += " , matching exclusion " + exclusion.matching;
     }
 
+    if (maybeLeakCause) {
+      string += "\n│                   ";
```

Comment:
Would be more efficient if you had used the string builder here as well

Reply:
Planning to convert module into Kotlin which will address the string concat here.

Reply:
👍 also we should probably move all this rendering into one neat separate class instead of being spread over several